vividus-plugin-web-app/src/main/java/org/vividus/bdd/steps/ui/web/ElementSteps.java

```diff
@@ -57,8 +56,7 @@
 public class ElementSteps implements ResourceLoaderAware
 {
     private static final String AN_ELEMENT_TO_CLICK = "An element to click";
-    private static final String THE_FOUND_ELEMENT_IS = "The found element is ";
-    private static final String AN_ELEMENT_WITH_THE_NAME = "An element with the name '%1$s'";
+    private static final String AN_ELEMENT_WITH_ATTRIBUTES = "An element with attributes%1$s";
```

Comment:
no space after 'attributes'?

Reply:
SearchAttributes toString() adds space at the beginning of string, so in that case additional space is not required

Reply:
got it